docs/rules-new-cards.md

```diff
@@ -128,11 +126,11 @@ Ten cards will be added each week, but there are restrictions on the number of c
 
 Attempts at mechanics in this section will likely fail for technical reasons:
 
-1.  Manually triggering "hook" abilities like summon, entomb, or active abilities
-2.  Replacement effects
+1.  Replacement effects
```

Comment:
i actually fucked up here, the proper way to do a numbered list in markdown is to start them all with `1.` so that other lines of code don't have to change. it accepts any number followed by a period though.

Reply:
https://riptutorial.com/markdown/example/1805/numbered-lists